.github/workflows/package_for_release.yml

```diff
@@ -145,7 +145,42 @@ jobs:
           - name: 📤  Upload image
             run: docker push ghcr.io/software-mansion/smelter:${{ github.sha }}
 
-    docker-with-web-renderer:
+    docker-arm64:
+      needs: docker-amd64
+      runs-on: ubuntu-latest
+      steps:
+          - name: 📥 Checkout repo
+            uses: actions/checkout@v4
+
+          - name: 🛠️ Setup Docker
+            run: |
+              docker run --privileged --rm tonistiigi/binfmt --install all
+              echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u ${{ github.actor }} --password-stdin
+
+          - name: 🎣 Pull Smleter AMD64 image
+            run: |
+              docker pull ghcr.io/software-mansion/smelter:${{ github.sha }}
+              echo "AMD64_IMAGE_ID=$(docker inspect --format='{{index .RepoDigests 0}}' ghcr.io/software-mansion/smelter:${{ github.sha }})" >> "$GITHUB_ENV"
+
+          - name: 🔨 Build Smelter ARM64 image
+            run: |
+              docker buildx build --platform linux/arm64 -t ghcr.io/software-mansion/smelter:${{ github.sha }} -f build_tools/docker/slim.Dockerfile .
+
+          - name: ⏩ Push ARM64 image
+            run: |
+              docker push ghcr.io/software-mansion/smelter:${{ github.sha }}
+              echo "ARM64_IMAGE_ID=$(docker inspect --format='{{index .RepoDigests 0}}' ghcr.io/software-mansion/smelter:${{ github.sha }})" >> "$GITHUB_ENV"
```

Comment:
Is this command show digest of a image that was build last? Seems hacky is there a way to avoid relaying on that build order?

Reply:
Discussed outside of GitHub